editor/source/Canvas.js

```diff
@@ -92,6 +95,10 @@ export default class Canvas {
     this._gridGraphic.onMouseMove(event, this._ctx)
   }
 
+  _bodyOnMouseUp(event) {
+    this._gridGraphic.bodyOnMouseUp(event, this.ctx)
```

Comment:
Could you test the `event.target.id` here instead of at the top of `this._gridGraphic`?